sdk/digitaltwins/Azure.DigitalTwins.Core/tests/config/common.config.json

```diff
@@ -1,5 +1,3 @@
 ﻿{
-    "MicrosoftTenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
-    "DigitalTwinsInstanceHostName": "https://common.api.scus.pp.azuredigitaltwins-ppe.net",
     "TestMode": "Playback"
```

Comment:
So the plan is to keep this config file, but move all the DT instance and auth info to the env file?